graphql/resolvers.go

```diff
@@ -395,17 +395,15 @@ func (r *queryResolver) Hosts(ctx context.Context, hostID *string, distroID *str
 func (r *queryResolver) Host(ctx context.Context, hostID string) (*restModel.APIHost, error) {
 	host, err := host.GetHostByIdWithTask(hostID)
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error Fetching host: %s", err.Error()))
+		return nil, ResourceNotFound.Send(ctx, err.Error())
```

Comment:
i think this one can remain `InternalServerError` since this err could potentially be a number of things?

Reply:
put it back